keras_nlp/models/mistral/mistral_backbone.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_nlp.api_export import keras_nlp_export
+from keras_nlp.backend import keras
+from keras_nlp.backend import ops
+from keras_nlp.layers.modeling.reversible_embedding import ReversibleEmbedding
+from keras_nlp.models.backbone import Backbone
+from keras_nlp.models.mistral.mistral_layer_norm import (
+    MistralLayerNormalization,
+)
+from keras_nlp.models.mistral.mistral_transformer_decoder import (
+    MistralTransformerDecoder,
+)
+
+
+def _mistral_kernel_initializer(stddev=0.02):
+    return keras.initializers.RandomNormal(stddev=stddev)
+
+
+@keras_nlp_export("keras_nlp.models.MistralBackbone")
+class MistralBackbone(Backbone):
+    def __init__(
```

Comment:
Let's definitely add detailed docstrings here (and all public symbols), but for the private features, can just leave a one-line type docstring `"""Attention layer for mistral models."""`.